### PR TITLE
Modifications on  Segmentation Example

### PR DESCRIPTION
**Pull request by charlieJ107**
Modified Tensorflow segmentation example from stock tensorflow to bigdl nano.

### CODE REVIEW

**Comment by charlieJ107** [comment]
Jakins passed http://10.112.231.51:18888/job/ZOO-PR-NanoTests/18/

File: python/nano/example/tensorflow/segmentation/Readme.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Quick Start 
+
+1. Setup Environment
+
+    Install the BigDL-Nano with Tensorflow support with `pip` in a conda environment. 
+
+    ```
+    pip install bigdl-nano[tensorflow]
+    ```
```
Comment (yangw1234) [inline python/nano/example/tensorflow/segmentation/Readme.md:13]: `source bigdl-nano-init`

File: python/nano/example/tensorflow/segmentation/segmentation.py [hunk 2]
```diff
@@ -39,6 +39,7 @@
 import tensorflow as tf
 
 from tensorflow_examples.models.pix2pix import pix2pix
+from bigdl.nano.tf.keras import Model as BigdlModel
```
Comment (yangw1234) [inline python/nano/example/tensorflow/segmentation/segmentation.py:42]: Let's just call it `Model`

**Comment by yangw1234** [comment]
LGTM

**Comment by yangw1234** [comment]
http://10.112.231.51:18888/job/ZOO-PR-NanoTests/19/ error is unrelated

**Comment by charlieJ107** [comment]
Test passed in http://10.112.231.51:18888/job/ZOO-PR-nano-test/33/

**Pull request merged**